ScreenManager.lua

```diff
@@ -533,6 +528,7 @@ end
 --
 function ScreenManager.registerCallbacks( callbacks )
     local registry = {};
+    local function null() end
```

Comment:
I wasn't sure of using either of this forms:
```lua
local null = function () end
local null = function() end
local null = function () end;
local null = function() end;
local function null() end
local function null() end;
```
But I considered this to be the most fitting to the coding style